compiler/plugins/target/AMD-AIE/air/CMakeLists.txt

```diff
@@ -138,7 +159,10 @@ iree_cc_library(
   SRCS
     "${IREE_MLIR_AIR_SOURCE_DIR}/lib/Util/Dependency.cpp"
     "${IREE_MLIR_AIR_SOURCE_DIR}/lib/Util/Util.cpp"
+    "${IREE_MLIR_AIR_SOURCE_DIR}/lib/Util/Outliner.cpp"
+    "${IREE_MLIR_AIR_SOURCE_DIR}/lib/Util/CostModel.cpp"
```

Comment:
These two files are needed by some air tiling passes in AIRTransformPass, as well as air-to-airrt pass.